package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-cli-bugsnag",
-  "version": "1.3.0",
+  "version": "2.0.0",
```

Comment:
Please leave out the version bump because we will do this as part of the deploy process. Agreed this should be a 'major' version change.